scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -511,15 +525,19 @@
                                         (constraints/make-fenzo-group-constraint
                                           db group #(guuid->considerable-cotask-ids (:group/uuid group)) running-cotask-cache))
                                       (:group/_job job)))))
-        needs-gpus? (constraints/job-needs-gpus? job)
         scalar-requests (reduce (fn [result resource]
-                                  (if-let [value (:resource/amount resource)]
-                                    (assoc result (name (:resource/type resource)) value)
-                                    result))
+                                  (let [value (:resource/amount resource)
+                                        type (:resource/type resource)]
```

Comment:
We should consider destructuring these above. You can do it with namespaced keywords too:

https://clojure.org/guides/destructuring#_namespaced_keywords

Reply:
done